test/key_stores/unencrypted_file_system_keystore.test.js

```diff
@@ -16,5 +18,20 @@ describe('Unencrypted file system keystore', () => {
         ctx.keyStore = new UnencryptedFileSystemKeyStore(KEYSTORE_PATH);
     });
 
+    it('test get key', async () => {
```

Comment:
nit: I think this is more or less covered in common test case for all key stores